extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1706,7 +1710,11 @@ static public function onRenderWhatLinksHereRow( &$row, &$level, &$defaultRender
 						->params( $titleData['wallTitleTxt'], $titleData['wallPageName'] )
 						->parse() .
 					' (' .
-					Linker::linkKnown( $wlhTitle, wfMessage( 'whatlinkshere-links' )->escaped(), [],  [ 'target' => $wfMsgOptsBase['articleTitle'] ]
+					Linker::linkKnown(
+						$wlhTitle,
+						wfMessage( 'whatlinkshere-links' )->escaped(),
+						[],
+						[ 'target' => $titleData['articleTitle'] ]
```

Comment:
The ```$wfMsgOptsBase``` var is not defined anywhere in this method, but ```$titleData``` is being initialized on L1703 in the same way I've seen ```$wfMsgOptsBase``` initialized in other functions.  What a shitshow.